src/routes/library.route.ts

```diff
@@ -4,14 +4,13 @@ import { cognitoMiddleware } from '../middleware/cognito.auth';
 
 export const library_route = Router();
 
-//secure: discuss, should we lock down the catalog?
-library_route.get('/', cognitoMiddleware, libraryController.searchForAlbum);
+library_route.get('/', cognitoMiddleware(), libraryController.searchForAlbum);
 
-//secure: mgmt
+// secure: mgmt
 library_route.post('/', cognitoMiddleware('station-management'), libraryController.addAlbum);
 
 //secure: mgmt
```

Comment:
just noting here that the comment says secure but it doesn't include the `station-management` arg

Reply:
in case it isn't clear, i'm talking about the GET for `/rotation` which i think correctly doesn't include the arg, but the comment indicates otherwise.

Reply:
Yes, it shouldn't be restricted to station management.